go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/ryanuber/columnize v2.1.0+incompatible
 	github.com/smira/go-xz v0.0.0-20150414201226-0c531f070014
 	github.com/spf13/cobra v0.0.5
-	github.com/stretchr/testify v1.5.0
+	github.com/stretchr/testify v1.5.1
```

Comment:
had to update because of https://github.com/stretchr/testify/pull/894

1.5.0 narrowed `suite.T()` to fake interface which doesn't expose full of `testing.T`, while I wanted to access `.Failed()` method